index.bs

```diff
@@ -1471,11 +1471,11 @@ object |transport|, run these steps.
 interface WebTransportError : DOMException {
   constructor(optional WebTransportErrorInit init = {});
 
-  readonly attribute octet? applicationProtocolCode;
+  readonly attribute (unsigned long long or bigint)? applicationProtocolCode;
 };
 
 dictionary WebTransportErrorInit {
-  octet applicationProtocolCode;
+  (([Clamp] unsigned long long) or bigint) applicationProtocolCode;
```

Comment:
Maybe we should always return bigint? It would avoid callers needing to special case depending on the return type.

Reply:
(Removed bigint)